pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -577,10 +517,11 @@ func (m *cgroupManagerImpl) Update(cgroupConfig *CgroupConfig) error {
 	}
 
 	unified := libcontainercgroups.IsCgroup2UnifiedMode()
+	var paths map[string]string
 	if unified {
 		libcontainerCgroupConfig.Path = m.Name(cgroupConfig.Name)
 	} else {
-		libcontainerCgroupConfig.Paths = m.buildCgroupPaths(cgroupConfig.Name)
```

Comment:
Because of:
https://github.com/opencontainers/runc/blob/master/libcontainer/cgroups/fs/fs.go#L284-L286